packages/kbn-shared-ux-components/src/toolbar/buttons/add_from_library/add_from_library.test.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { mount as enzymeMount, ReactWrapper } from 'enzyme';
+import React from 'react';
+import {
+  SharedUxServicesProvider,
+  SharedUxServices,
+  mockServicesFactory,
+} from '@kbn/shared-ux-services';
+import { ToolbarButton } from '../primary/primary';
+
+describe('<ToolbarButton />', () => {
+  let services: SharedUxServices;
+  let mount: (element: JSX.Element) => ReactWrapper;
+
+  beforeEach(() => {
+    services = mockServicesFactory();
+    mount = (element: JSX.Element) =>
```

Comment:
Do you need this? The button is not actually using any services?

Reply:
You're right [63d8c31](https://github.com/elastic/kibana/pull/127605/commits/63d8c31f0de1ff81e9cff47f29178a03f01fc9d7) for changes thanks!!